pkg/secretsencrypt/config.go

```diff
@@ -275,8 +276,14 @@ func GetEncryptionConfigMetrics(runtime *config.ControlRuntime, initialMetrics b
 			return true, fmt.Errorf("encryption reload time is incorrectly ahead of current time")
 		}
 
-		reloadSuccessCounter = int64(successMetric.GetMetric()[0].GetCounter().GetValue())
-
+		for _, totalMetric := range totalMetrics.GetMetric() {
+			logrus.Debugf("totalMetric: %+v", totalMetric)
+			for _, label := range totalMetric.GetLabel() {
+				if label.GetValue() == "success" {
```

Comment:
nit: technically shouldn't we check `label.GetName == "status"` as well?

Reply:
Addresses